src/packages/v4/components/V4SplitList.tsx

```diff
@@ -0,0 +1,24 @@
+import { BigNumber } from "@ethersproject/bignumber"
+import SplitList, { V2V3SplitListProps } from "packages/v2v3/components/shared/SplitList"
```

Comment:
IMO we should either figure out how to make this truly generic component (not v2v3-related), or just duplicate it and modify for v4. I think we should keep v4 and v2v3 as separated as possible.

Reply:
Addressed https://github.com/jbx-protocol/juice-interface/pull/4399#discussion_r1685956311